nixos/modules/services/web-apps/healthchecks.nix

```
@@ -186,7 +196,10 @@ in
           WorkingDirectory = cfg.dataDir;
           User = cfg.user;
           Group = cfg.group;
-          EnvironmentFile = [ environmentFile ];
+          EnvironmentFile = [
+            environmentFile
+            (lib.optional (cfg.settingsFile != null) cfg.settingsFile)
+          ];
```

`lib.optional` returns a list
```suggestion
          EnvironmentFile = [
            environmentFile
          ] ++ lib.optional (cfg.settingsFile != null) cfg.settingsFile;
```

---

Module was working as it was, but it does return I list. I've fixed this now.